src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -793,3 +797,198 @@ def plot_GP_components(lc, model, meta, fitter, isTitle=True):
         fig.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
         if not meta.hide_plots:
             plt.pause(0.2)
+
+
+def plot_fleck_star(lc, model, meta, fitter):
+    
+    # find number of spots
+    spotinds = [s for s in model.parameters.dict.keys() if 'spotrad' in s]   
+    counter = [s for s in spotinds if '_' in s]
+    nspots = len(spotinds) - len(counter)
+
+    # find number of limb darkening parametesr
+    limbdarks = [s for s in model.parameters.dict.keys() if 'u' in s] 
+    counter = [s for s in limbdarks if '_' in s]
+    nus = len(limbdarks) - len(counter)
```

Comment:
We already have code implemented to nicely handle limb-darkening parameters for the batman transit model - no need to do something different here

Reply:
Ah yeah, I wrote this before the AstroModel PlanetParams class became a thing and needed to find the limb darkening values (annoying that fleck needs them to plot up the star...). Will update